### PR TITLE
fix(responsive): Header menu was not collapsing

### PR DESCRIPTION
**Pull request by desoindx**

### CODE REVIEW

File: src/components/shared/layout/Header.tsx [hunk 1]
```diff
@@ -58,39 +61,19 @@ const Header = ({
...
-                className="fr-nav__link"
-                aria-current={currentMenu === url ? 'page' : undefined}
-              >
-                {label}
-              </a>
-            </Link>
-          </li>
+            title={label}
+            current={currentMenu === url}
+            asLink={<a href={url}></a>}
```
Comment (nkokla) [inline src/components/shared/layout/Header.tsx:68]: Should the `asLink`props value not be something like `<Link href  ...` ?

File: src/components/shared/layout/Header.tsx [hunk 2]
```diff
@@ -58,39 +61,19 @@ const Header = ({
...
-                }
-              >
-                Gestionnaire
-              </a>
-            </Link>
-          </li>
+          <NavItem
+            title="Gestionnaire"
+            current={currentMenu === '/gestionnaire'}
+            asLink={<a href="/gestionnaire"></a>}
```
Comment (nkokla) [inline src/components/shared/layout/Header.tsx:75]: Same here.

File: src/components/shared/layout/Header.tsx (hunk 1 above)
Comment (desoindx) [reply src/components/shared/layout/Header.tsx:68]: it should be :/ but it's not working because Link does not allow an onClick property

Comment (nkokla) [reply src/components/shared/layout/Header.tsx:68]: I don't see why we need an onClick ?
Also, With these links, we bypass the router and loose the construction of the unique layout (each click on menu will rebuild all the page content, so we loose the interrest of the SPA) and we can not use the Context update off the page (the content load in context after build of the layout will be lost at each rebuild).

Comment (desoindx) [reply src/components/shared/layout/Header.tsx:68]: to close the menu when changing menu in mobile view

**Pull request merged**